christfellowshipchurch/web-app-v2
# Background videos

### About
This PR allows us to pull in videos from Wistia and removes hardcoded videos to save bandwidth. 

### Test Instructions
1. Go to `/locations` 
2. Click on each of the location pages. The videos should play correctly on desktop and mobile. 

### Screenshots
<img width="1154" alt="Screenshot 2023-07-06 at 2 27 11 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/77253a1b-097a-4235-9bab-ad996edcec96">


### Closes Tickets
[CFDP-2618](https://christfellowshipchurch.atlassian.net/browse/CFDP-2618)


[CFDP-2618]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
